examples/00_quick_start/fbt_movielens.ipynb

```diff
@@ -0,0 +1,770 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    sys.path.append("../../")
</pre><p>in the latest version of staging we removed the append, we are publishing a pip package so just having import reco_utils should work. You can install as <code>pip install -e .[all]</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1457/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Addressed the sys path setup.</p>